caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -73,11 +77,17 @@ class SqlEditorTopToolbar extends React.Component {
   }
   fetchDatabaseOptions() {
     this.setState({ databaseLoading: true });
-    const url = '/databaseasync/api/read';
+    const url = '/databaseasync/api/read?_flt_0_expose_in_sqllab=1';
```

Comment:
I didn't find the _flt_0_expose_in_sqllab arg, could you please explain it in the comment ?